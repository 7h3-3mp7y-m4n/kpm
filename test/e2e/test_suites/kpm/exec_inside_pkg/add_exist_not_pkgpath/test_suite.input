kpm add /path/to/not/exist