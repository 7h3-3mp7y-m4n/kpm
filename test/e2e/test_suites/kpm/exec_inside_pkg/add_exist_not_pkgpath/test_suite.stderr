failed to select latest version from 'localhost:5001/test/path/to/not/exist'
